### DISCUSSION TITLE
clicking view changes creates a Git error

### DISCUSSION THREAD

**Discussion by seraklab** [ProjectBorealis/UEGitPlugin#137] [General]
If I click on view changes

![image](https://github.com/ProjectBorealis/UEGitPlugin/assets/133740034/6882b722-dd56-43a5-a6be-c365b3b30b59)

I get prompted that a git command failed

![image](https://github.com/ProjectBorealis/UEGitPlugin/assets/133740034/8efb5f8c-375c-4f1d-8fbc-f83cbcb0fbd0)

In the log I am told that UpdateStatus Failed, the rest of the plugin is working fine though

![image](https://github.com/ProjectBorealis/UEGitPlugin/assets/133740034/880c94ec-31f5-46a1-8f4c-b988ba6a8342)

**Comment by LuryannC** [comment]
I am having the same thing.
